tests/test_plugin.py

```diff
@@ -22,3 +33,33 @@ def test_anon(client):
     response = client.get(url_for('Airflow.index'))
     assert response.status_code == 302
     assert b"update-notice.css" not in response.data, "Don't show notice when logged out"
+
+
+@pytest.mark.parametrize(
+    "columns",
+    [('end_of_support', 'eos_dismissed_until')],
```

Comment:
Should we refactor so we can get the list of columns from `plugin.py`, or maybe from `models.py`?

Reply:
Changed it